doc/setting_up_vcpkg.md

```diff
@@ -2,6 +2,43 @@
 
 This document describes how to setup vcpkg to build applications using Microsoft Azure IoT device C SDK. It demonstrates building and running a C language SDK sample for Windows, Linux, and Mac.
 
+## C SDK vcpkg support
+
+The C SDK uses [vcpkg](https://github.com/microsoft/vcpkg) primarily for LTS releases. The most recent LTS release can be installed using the syntax: `vcpkg install azure-iot-sdk-c` and builds using the following CMake flags:
+
+```
+    -Dskip_samples=ON
+    -Duse_installed_dependencies=ON
+    -Duse_default_uuid=ON
+    -Dbuild_as_dynamic=OFF
+    -Duse_edge_modules=ON
+    -Dwarnings_as_errors=OFF
+```
+
+The `use-prov-client` feature uses the syntax: `vcpkg install azure-iot-sdk-c[use-prov-client]` and builds using additional CMake flags:
+```
+    -Dhsm_type_symm_key=ON
+    -Duse_prov_client=ON
+```
+
+The `public-preview` feature allows you to build and install the azure-iot-sdk-c from a provided public-preview hash. The Ref hash used can be found in the [portfile.cmake](https://github.com/microsoft/vcpkg/blob/master/ports/azure-iot-sdk-c/portfile.cmake) file. To access this feature, use the syntax: `vcpkg install azure-iot-sdk-c[public-preview]`. The following CMake flags will be set:
+
+```
+    -Dskip_samples=ON
+    -Duse_installed_dependencies=ON
+    -Duse_default_uuid=ON
+    -Dbuild_as_dynamic=OFF
+    -Duse_edge_modules=ON
+    -Dwarnings_as_errors=OFF
+```
+
+There are no other features available.
+
+> NOTE: If your application requires specific CMake flags not shown above, please build and install directly from the source code.  See [devbox_update.md](https://github.com/Azure/azure-iot-sdk-c/blob/main/doc/devbox_setup.md) for further information.
```

Comment:
```suggestion
> NOTE: If your application requires specific CMake flags not shown above, please build and install directly from the source code.  See [devbox_setup.md](https://github.com/Azure/azure-iot-sdk-c/blob/main/doc/devbox_setup.md) for further information.
```

Reply:
oh crazy.  did i totally mistype that, or did the name change?